README.md

```diff
@@ -337,6 +347,15 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: Check out the [Globo website](https://hacktoberfest.globo.com/) for more details.
 - **Notes**: Only the first 100 people residing in Brazil will receive the t-shirt.
 
+#### **MayaData & OpenEBS**(Special swags ,Oculus VR Headset)
+
+- **Swag**: Special swags ,Oculus VR Headset
```

Comment:
Instead of "Special swags", can you change it to t-shirt and stickers? Also, the VR headset is only for the Top Contributor, so  can you please add an entry in that section of the List as well?

Also, this section is not just a copy-and-paste from the A-Z section. Be sure to check the [Contributing.md file](https://hacktoberfestswaglist.com/contributing) for how to format this section.